apps/wallet-mobile/src/features/Receive/useCases/ReceiveScreen.tsx

```diff
@@ -0,0 +1,126 @@
+import {useFocusEffect} from '@react-navigation/native'
+import {useTheme} from '@yoroi/theme'
+import _ from 'lodash'
+import React from 'react'
+import {SafeAreaView, ScrollView, StyleSheet, View} from 'react-native'
+
+import Icon from '../../../assets/img/copy.png'
```

Comment:
SVG no pngs

Reply:
But button expect only .png and .jpeg images

Reply:
cc: @SorinC6